crates/ruff_formatter/src/format_element/document.rs

```diff
@@ -450,8 +450,8 @@ impl Format<IrFormatContext<'_>> for &[FormatElement] {
                             )?;
                         }
 
-                        StartLineSuffix => {
-                            write!(f, [text("line_suffix(")])?;
+                        StartLineSuffix(_) => {
+                            write!(f, [text("line_suffix(")])?; // TODO(cnpryer): Would it be useful to have this info in the IR display?
```

Comment:
Maybe `"line_suffix<width>("`

Reply:
The most common pattern is to write the property keys first

```
conditional_group(condition: if_group_breaks("#1"), [
  conditional_group(condition: if_group_breaks("#1"), [
    fits_expanded(propagate_expand: true, condition: if_group_fits_on_line("#1"), [
      group(expand: propagated, [
```

https://play.ruff.rs/81156166-0f8b-4f66-b66e-ca5039fca5b1?secondary=FIR

Reply:
Added using `dynamic_text` in 266c0695d434560e0cb6f55083ebf4ded99ae84a. I'll mark this as resolved, but if there's a better way to do it than `std::format!` and the pattern I used I can revise.